### PR TITLE
Update to 2.5.2

### PR DESCRIPTION
**Pull request by pvuorela**
Noted on the make check removal commit that I had one x86_64 failure locally when building 2.5.1 (for arm). Not sure about this newer version, but either way thinking we wouldn't need to worry about that too much now.

@abranson @spiiroin @okodron

### CODE REVIEW

**Comment by Thaodan** [comment]
The host arch should not be visible at all in sb2. I think disabling those checks entirely is bad. You should check if the test also breaks on obs.

The test don't run on device but on the builder.

**Comment by pvuorela** [comment]
> I think disabling those checks entirely is bad

Why? This is a library used currently by a single project. Do we gain enough from it to warrant the build time increase and possible false positives in case the sb2 build environment doesn't match the expectations?

**Comment by pvuorela** [comment]
FWIW same single x86_64 failure on 2.5.2 locally.

File: rpm/libseccomp.spec [hunk 1]
```diff
@@ -42,7 +42,7 @@ for libseccomp.
 %build
 %autogen
 %configure --disable-static
```
Comment (okodron) [inline rpm/libseccomp.spec]: Can be replaced with
```suggestion
%reconfigure --disable-static
```

**Comment by spiiroin** [comment]
> The host arch should not be visible at all in sb2.

First thought: My understanding is that seccomp is fairly advanced kernel thing. Can we even expect that it works in sb2 environment sufficiently well to run all those tests / do positive test results even mean that things are ok also from real target device point of view?

**Review by spiiroin — APPROVED** [review]
I might be wrong but I'm inclined to think that extensive kernel feature testing in sb2 env is more or less pointless and thus skipping such tests during build time is ok.

**Comment by Thaodan** [comment]
> I might be wrong but I'm inclined to think that extensive kernel feature testing in sb2 env is more or less pointless and thus skipping such tests during build time is ok.

That makes sense and would explain why the host arch leaks through in the test. We need to look more into those kind of things regarding unit-tests.

File: rpm/libseccomp.spec (hunk 1 above)
Comment (pvuorela) [reply rpm/libseccomp.spec]: Changed

**Pull request merged**